.github/workflows/conda-python-build.yaml

```diff
@@ -0,0 +1,62 @@
+name: conda-python-build
+
+concurrency:
+  group: conda-python-build-on-${{ github.event_name }}-from-${{ github.ref_name }}
+  cancel-in-progress: true
+
+on:
+  # run only when called by other workflows
+  workflow_call:
+    inputs:
+      script:
+        required: true
+        type: string
+        description: "relative path to a script that builds conda packages"
+
+env:
```

Comment:
everything below here is identical to what was in `github-actions.yml`, except that the literal `ci/build_python.sh` has been replaced by `${{ input.script }}`, so that we have the option to use different scripts in the future.